question16/main.js

```diff
@@ -83,21 +82,57 @@ const createTitles = ({ articles }) => {
     const titleList = document.createElement('li');
     const articleLink = document.createElement('a');
     articleLink.href = "#";
-    articleLink.textContent = articleTitle; 
+    articleLink.textContent = articleTitle;
+
+    // Display new icon
+    const elapsedTime = getElapsedDays(articles[i]);
+    if (elapsedTime <= 3) {
+      const newIcon = createElementWithClassName("span", "new_icon"); 
+      newIcon.textContent = 'new';
+      articleLink.appendChild(newIcon);      
+    }
+
+    //add comment icon and number
+    const commentsLength = articles[i].comments.length; 
+    if (commentsLength > 0) {
+      articleLink.appendChild(createCommentIcon(articles[i]));
+    }
```

Comment:
I added here. This is a bit of a long function here, so I was wondering if I could write it more clearly.